tcf_core/settings/ci.py

```diff
@@ -0,0 +1,13 @@
+# pylint: disable=unused-wildcard-import,wildcard-import,fixme
+"""
+Django settings module for CI services such as Travis or GitHub Actions
+
+Unlike other files where we use environment variables for everything, it is
+okay to use some strings constants here and in .travis.yml or python-app.yml
+"""
+from .base import *
+
+# The following if statement is needed to prevent overwriting global variables when
+# this settings file is interpreted
+if os.environ.get('DJANGO_SETTINGS_MODULE') == 'tcf_core.settings.ci':
+    pass
```

Comment:
Uhh, it just so happens that `settings/ci.py` coincides with `settings/base.py` because environment variables were enough to handle all environment-specific differences, but the `if` statement can be used to make any changes to Django settings without using environment variables. (If this `if` statement were absent, future maintainers will encounter the same bug that I experienced.) Also, it makes the settings modules consistent.